caddyhttp/proxy/proxy.go

```diff
@@ -197,10 +218,10 @@ func (p Proxy) ServeHTTP(w http.ResponseWriter, r *http.Request) (int, error) {
 		timeout := host.FailTimeout
 		if timeout > 0 {
 			atomic.AddInt32(&host.Fails, 1)
-			go func(host *UpstreamHost, timeout time.Duration) {
+			go func() {
 				time.Sleep(timeout)
 				atomic.AddInt32(&host.Fails, -1)
-			}(host, timeout)
+			}()
```

Comment:
I'll revert it. I changed it because it was not written in the same style as other parts of the code, which all use implicit binding of closure parameters.